app/src/main/java/us/shandian/giga/get/DownloadMissionRecover.java

```diff
@@ -131,31 +132,38 @@ private void resolveStream() throws IOException, ExtractionException, HttpError
 
         switch (mRecovery.getKind()) {
```

Comment:
Have you checked if it's ok for this method to return `null` (either an exception is manually thrown or the case is handled, by the caller)?

Reply:
That's not my code, I don't think we should change it in this PR.